Tools/TestWebKitAPI/Tests/WebKitCocoa/SiteIsolation.mm

```diff
@@ -2616,24 +2616,50 @@ HTTPServer server({
         },
     });
 
-    [webView evaluateJavaScript:
-        @"let i = document.getElementById('testiframe');"
-        "i.addEventListener('load', () => { alert('iframe loaded') });"
-        "i.src = 'https://webkit.org/never_respond';"
-        "setTimeout(()=>{ i.src = 'https://example.com/respond_quickly' }, Math.random() * 500)"
-        completionHandler:nil];
-    EXPECT_WK_STREQ([webView _test_waitForAlert], "iframe loaded");
-    checkFrameTreesInProcesses(webView.get(), {
+    auto checkStateAfterSequentialFrameLoads = [webView = RetainPtr { webView }, navigationDelegate = RetainPtr { navigationDelegate }] (NSString *first, NSString *second, Vector<ExpectedFrameTree>&& expectedTrees) {
```

Comment:
`webView` and `navigationDelegate` are already RetainPtr, so we could just capture them by value?

Reply:
I like that better too, but clang on Ventura doesn't compile that successfully since they are instantiated using structured bindings.  This is indeed needed to compile on Ventura.